src/kvp/label/mod.rs

```diff
@@ -305,4 +278,25 @@ impl Labels {
 
         Ok(labels)
     }
+
+    delegate! {
```

Comment:
Maybe it's just me but it's not obvious what this is doing. I have no idea what "delegate!" is.
If in doubt, add a comment

Reply:
Sure. I will add a comment. It basically forwards the impl to the inner type.

Reply:
You don't have to start a new PR for this.
It was more a general remark: Help your future self and other future people with logging and comments.

Comments should (in my opinion) mostly about the "why" not the "what", sometimes - if it's not obvious - about the "how this works"

Reply:
This should now be resolved by 20274e7